src/wovnio/wovnphp/Store.php

```diff
@@ -260,4 +260,15 @@ public function defaultLang()
     {
         return $this->settings['default_lang'];
     }
+
+    /**
+     * Tests if a lang code is a valid lang code or lang alias.
+     *
+     * @param $code String the lang code to be tested
+     * @return bool if code is a valid Lang or Lang alias
+     */
+    public function isLangOrLangAlias($code)
```

Comment:
I could't find out the place where this method is used.

Reply:
Sorry totally missed this one. I added this one thinking it might be useful during the refactoring process, but didn't end up using it. I'll remove this one.